packages/bundle/src/__tests__/addMetaTag/full.spec.js

```diff
@@ -0,0 +1,49 @@
+/** @jest-environment @happy-dom/jest-environment */
+
+describe('importing full bundle', () => {
+  beforeEach(() => {
+    jest.spyOn(console, 'warn').mockImplementation(() => {
+      // Intentionally left blank.
+    });
+
+    // "adaptivecards" is trying to load "swiper", which is ESM only, mocking it
+    jest.mock('swiper', () => ({}));
+    jest.requireActual('../../../dist/botframework-webchat.js');
+  });
+
+  test('should have injected <meta> tag', () => {
+    const metaMap = new Map(
+      Array.from(document.querySelectorAll('meta'))
+        .filter(({ name }) => name.startsWith('botframework-webchat'))
+        .map(({ content, name }) => [name, content])
+    );
+
+    expect(metaMap.has('botframework-webchat:api')).toBe(true);
+    expect(metaMap.has('botframework-webchat:bundle')).toBe(true);
+    expect(metaMap.has('botframework-webchat:component')).toBe(true);
+    expect(metaMap.has('botframework-webchat:core')).toBe(true);
+
+    expect(
+      metaMap
+        .get('botframework-webchat:api')
+        .split(';')
+        .map(value => value.trim())
+    ).toEqual(expect.arrayContaining(['build-tool=tsup', 'module-format=esmodules']));
```

Comment:
Consider setting this only for actual `.mjs` (don't see it tested though)

Reply:
Removed.

When `bundle` is importing `api`, `jest-babel` will try to load CJS version of `api`. Tried few things but I cannot force it to load ESM version and transpile it.

We would rely on testing it using some other mechanism.